POS/Operation/04_CompletePurchase.md

```diff
@@ -2,21 +2,232 @@
+    The reduced price of the purchase is displayed below the percentage field.
+
+5. Click the [APPLY DISCOUNT] button in the bottom right corner.   
+    The bill discount view is closed. The pay bill list is displayed again. The discount is indicated both as a percentage and as a fixed amount above the reduced total.
+
+    ![Pay Discount](../../Assets/Screenshots/POS/Sales/Cashpoint/BillList/PayDiscount.png "[Pay Discount]")
+
+    > [Info] To the right of the indicated discount, on the edge of the bill list, the ![Remove](../../Assets/Icons/Cross06.png "[Remove]") (Remove) flag is displayed. Click the flag to delete the discount, if required.
+
+
+<!--- NEU-->
+## Check the customer-specific discount
+
+If your company uses the *Customer-specific discount* feature, a specific discount might be defined for a customer. The discount is automatically deducted as soon as a customer purchases an offer. You can remove the discount, if required.
+
+> [Info] If a customer has a customer-specific discount, other discount options are not allowed.
+
+#### Prerequisites
+
+- Offers for POS are created, see [Manage offers for POS](../Integration/07_ManageOffers.md).
+- At least one offer is added to the bill list, see [Select an offer](./02_SelectOffer.md#select-an-offer).
+- The *Customer-specific discount* setting is activated. For detailed information, see [Customer-specific discount](../UserInterface/02a_GlobalSettings.md#customer-specific-discount).
+
+#### Procedure
+
+*Venduo POS > Sales > Select store and pay desk > Tab CASHPOINT*
+
+![Cashpoint](../../Assets/Screenshots/POS/Sales/Cashpoint/Product.png "[Cashpoint]")
+
+1. Add an existing customer to the bill list. For detailed information, see [Assign a purchase to an existing customer](../Operation/03_EditBillList.md#assign-a-purchase-to-an-existing-customer)
+
+2. Click the [PAY] button at the bottom of the bill list.   
+    The pay bill list is displayed. A discount is added automatically to the pay bill list. 
+
+    ![Check discount](../../Assets/Screenshots/POS/Sales/Cashpoint/CustomerSpecificDiscount.png)
+
+3. If the customer should not receive a discount, click the ![Remove](../../Assets/Icons/Cross06.png "[Remove]") (Remove) flag.    
+    The customer-specific discount is removed.
+
+
+<!---NEU-->
+## Assign a price group
+
+A price group classifies different prices that apply to customers to whom the same pricing criteria are applied, for example club members or business partners. For example, an assigned price group can reduce the price for a specific offer. The price group is stored in the customer master data, so that some offers may cost less.   
+
+If your company supports this feature, the price will be automatically reduced for the customers concerned. You can also assign the price group manually. 
+
+#### Prerequisites
+
```

Comment:
At least one offer is added to the bill list, see [Select an offer](Link).

Braucht man das hier nicht? Ausgangssituation (Screenshot) deutet so an.